tests/figure.spec.ts

```diff
@@ -10,11 +10,8 @@ test('can open a simple figure', async ({ page }) => {
   const figureName = generateVarName();
 
   // Create a figure that uses the table we just created
-  await page.keyboard.type(`from deephaven import empty_table\r`);
-  await page.keyboard.type(`from deephaven.plot.figure import Figure\r`);
-  await page.keyboard.type(
-    `${figureName} = Figure().plot_xy(series_name="Test", t=empty_table(100).update(["x=i", "y=Math.sin(i)", "z=Math.cos(i)"]), x="x", y="y").show()\r`
-  );
+  const command = `from deephaven import empty_table\rfrom deephaven.plot.figure import Figure\r${figureName} = Figure().plot_xy(series_name="Test", t=empty_table(100).update(["x=i", "y=Math.sin(i)", "z=Math.cos(i)"]), x="x", y="y").show()\r`;
```

Comment:
Probably makes sense to do `\n` instead of `\r` as people are more familiar with that. In fact, it would look better to just re-write it with a multi-line string, e.g. (note that white space matters):
```
const command = `
from deephaven import empty_table
from deephaven.plot.figure import Figure
${figureName} = Figure().plot_xy(series_name="Test", t=empty_table(100).update(["x=i", "y=Math.sin(i)", "z=Math.cos(i)"]), x="x", y="y").show()
`;
```